lib/ecto_enum/postgres.ex

```diff
@@ -40,8 +40,8 @@ defmodule EctoEnum.Postgres do
         def __valid_values__(), do: @valid_values
 
         def create_type() do
-          types = Enum.join(unquote(list), ", ")
-          sql = "CREATE TYPE #{unquote type} AS ENUM (#{types})"
+          types = Enum.join(unquote(list), "', '")
+          sql = "CREATE TYPE #{unquote type} AS ENUM ('#{types}')"
```

Comment:
Didn't realise there was an ecto-2.0 branch, so ended up making the same change haha. Anyway just wanted to point out this line 43 could instead be simplified as just: `types = Enum.map_join(unquote(list), ", ", &("'#{&1}'"))`. And so not needing to change line 44.